docs/networks/node-ops/access-onchain-data/access-nodes/accessing-data/access-api.md

```diff
@@ -327,6 +381,109 @@ message TransactionResultResponse {
   repeated flow.Event events
 }
 ```
+### GetTransactionResultByIndex
+
+`GetTransactionResultByIndex` gets the execution result of a transaction at a specified block and index.
+
+```proto
+rpc GetTransactionResultByIndex(GetTransactionByIndexRequest) returns (TransactionResultResponse);
+```
+
+#### Request
+
+```proto
+message GetTransactionByIndexRequest {
+  bytes block_id = 1;
+  uint32 index = 2;
```

Comment:
This field is likely missing from other endpoints as well. Can you compare the docs to the protobuf to see if there are any other args missing?
```suggestion
  uint32 index = 2;
  entities.EventEncodingVersion event_encoding_version = 3;
```